stl/src/filesystem.cpp

```diff
@@ -168,6 +169,10 @@ namespace {
 
         return __std_win_error{GetLastError()};
     }
+
+    [[nodiscard]] static unsigned long long _Merge_to_ull(unsigned long _High, unsigned long _Low) noexcept {
+        return static_cast<unsigned long long>(_Low) | (static_cast<unsigned long long>(_High) << 32);

```

Comment:
Style: I recommend swapping the bitwise-OR's operands to match the (now big-endian) parameter order.